BREAKING.md

```diff
@@ -10,6 +12,9 @@
 - [Creating new containers with Container.load is no longer supported](#Creating-new-containers-with-Containerload-is-no-longer-supported)
 - [getHashedDocumentId is now async](#gethasheddocumentid-is-now-async)
 
+### Changes to dataStoreRuntime and dataStoreContext interfaces
+- `documentId` property is removed from dataStoreRuntime and dataStoreContext interfaces. It should not be exposed at data store level.
```

Comment:
Consider saying something like - It is a document level concept and is not exposed from data store level anymore.